docs/installation.rst

```diff
@@ -53,6 +53,44 @@ or for different functionalities of Modin. Here is a list of dependency sets for
 
   pip install "modin[dask]" # If you want to use the Dask backend
 
+Installing with conda
+---------------------
+
+Modin releases can be installed using ``Miniconda`` from conda-forge channel. Starting from 0.10.1
+it is possible to install modin with chosen engine(s). Current options are:
+
++---------------------------------+---------------------------+
+| **Package name in conda-forge** | **Engine(s)**             |
++---------------------------------+---------------------------+
+| modin                           | Dask                      |
++---------------------------------+---------------------------+
+| modin-all                       | Dask, Ray, OmniSci_       |
++---------------------------------+---------------------------+
+| modin-dask                      | Dask                      |
++---------------------------------+---------------------------+
+| modin-ray                       | Ray                       |
++---------------------------------+---------------------------+
+| modin-omnisci                   | OmniSci                   |
```

Comment:
```suggestion
| modin-omnisci                   | Ray, OmniSci_              |
```
to make it a link as well

also maybe other engines should have a link?..

Reply:
I made such link in modin-all line

Reply:
Added link for other engines